playlistmanager.lua

```diff
@@ -663,10 +661,63 @@ function draw_playlist()
 	
   ass:append(settings.style_ass_tags)
 
-  -- TODO: padding should work even on different osd alignments
-  if mp.get_property("osd-align-x") == "left" and mp.get_property("osd-align-y") == "top" then
-    ass:pos(settings.text_padding_x, settings.text_padding_y)
+  -- align from mpv.conf
+  local align_x = mp.get_property("osd-align-x")
+  local align_y = mp.get_property("osd-align-y")
+  -- align from style_ass_tags
+  if settings.style_ass_tags ~= nil then
+    local an_tag = settings.style_ass_tags:match('\\an%d')
```

Comment:
You can do a capture group `match('\\an(%d)')` to avoid 2 matches and extra level of indentation.

```lua
local an = tonumber(settings.style_ass_tags:match('\\an(%d)'))
if an == 1 then
  align_x = 'left'
  align_y = 'bottom'
elseif an == 2 then
...
```